### ISSUE TITLE
add legend of the weather map

### ISSUE THREAD

**Issue by letypequividelespoubelles** [osmandapp/OsmAnd#16072]
# 🚀 feature request

### Description
When on the weather map (for exemple temperature map), we don't gave the legend / signification of the color.

### Describe the solution you'd like

Add a legend for the weather map.

### Describe alternatives you've considered
 
None

Thanks a lot for this amazing app

**Comment by vshcherb** [comment]
Already present in documentation - https://docs.osmand.net/docs/user/plugins/weather#weather-layers

**Comment by letypequividelespoubelles** [comment]
Yes, i saw it,but it would be better to have on the same moment, at the bottom of the screen would be imho great. And if we want it to be perfect, it should be adaptable legend, with the min and Max set to the min Max displayed on this area!

**Comment by dantecatalfamo** [comment]
This is quite confusing as a new user. It's not obvious that you need to check the documentation for the legend
